src/components/EBox.vue

```diff
@@ -0,0 +1,43 @@
+<!-- @TODO: better component naming -->
+<script setup lang="ts">
+type Props = {
+  el?: "div" | "article" | "section" | string;
+  ratio?:
+    | "16 / 9"
+    | "4 / 3"
+    | "1 / 1"
+    | "3 / 4"
+    | "9 / 16"
+    | "1 / 2"
+    | "2 / 1"
+    | "auto";
+  color?: "accent" | "gray";
```

Comment:
Lisa `TODO Rethink how to pass colors to components`. Ma arvan et voiks olla voimalik pass'ida `var(--blue-500)` naiteks aga see on eraldi laiem teema.